pkg/cmd/server/api/validation/validation.go

```diff
@@ -69,6 +70,35 @@ func ValidateKubeConfig(path string, field string) errs.ValidationErrorList {
 	return allErrs
 }
 
+func ValidateKubernetesMasterConfig(config *api.KubernetesMasterConfig) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+
+	if len(config.MasterIP) > 0 {
+		allErrs = append(allErrs, ValidateSpecifiedIP(config.MasterIP, "masterIP")...)
+	}
+
+	if len(config.ServicesSubnet) > 0 {
+		if _, _, err := net.ParseCIDR(strings.TrimSpace(config.ServicesSubnet)); err != nil {
+			allErrs = append(allErrs, errs.NewFieldInvalid("servicesSubnet", config.ServicesSubnet, "must be a valid CIDR notation IP range"))
```

Comment:
Why not report the actual error with advice?  Not everyone happens to know CIDR notation. :)